### PR TITLE
query search bar

### PR DESCRIPTION
**Pull request by vunguyen2712**

### CODE REVIEW

File: app/app-config.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ function appConfig($urlRouterProvider, $httpProvider) {
     $urlRouterProvider.when('/', '/home');
     $httpProvider.defaults.useXDomain = true;
     delete $httpProvider.defaults.headers.common['X-Requested-With'];
+    $urlRouterProvider.when('/questions', '/home');
```
Comment (ghost) [inline app/app-config.js]: duplicate line.

Comment (vunguyen2712) [reply app/app-config.js]: removing

**Comment by ghost** [comment]
Pull in release-next. I see old code here.

File: app/home/home-controller.js [hunk 2]
```diff
@@ -1,4 +1,40 @@
 function homeController() {
+
+    // var video = document.getElementById('video');
```
Comment (ghost) [inline app/home/home-controller.js]: remove this

**Review by ghost — APPROVED** [review]

**Review by Stephyraad — APPROVED** [review]

**Pull request merged**